src/test/java/com/openlattice/rehearsal/authentication/MultipleAuthenticatedUsersBase.java

```diff
@@ -81,9 +81,9 @@
 import retrofit2.Retrofit;
 
 public class MultipleAuthenticatedUsersBase extends SetupEnvironment {
-    private final static Map<String, Retrofit> retrofitMap = new HashMap<>();
-    private final static Map<String, Retrofit> linkerRetrofitMap = new HashMap<>();
-    private final static Map<String, OkHttpClient> httpClientMap = new HashMap<>();
+    private final static Map<String, Retrofit> retrofitMap = new HashMap<>(5);
```

Comment:
I hear `Maps.newHashMapWithExpectedSize( blah )` is more performant :)

Reply:
I hear `ImmutableMap.of()` is all the rage these days 🚛